pkg/common/events.go

```diff
@@ -56,62 +58,109 @@ var ValidEvents = []gh.Event{
 	gh.ProjectEvent,
 	gh.PublicEvent,
 	gh.PullRequestEvent,
-	gh.PullRequestReviewEvent,
 	gh.PullRequestReviewCommentEvent,
+	gh.PullRequestReviewEvent,
 	gh.PushEvent,
 	gh.ReleaseEvent,
 	gh.RepositoryEvent,
+	gh.RepositoryVulnerabilityAlertEvent,
+	gh.SecurityAdvisoryEvent,
 	gh.StatusEvent,
-	gh.TeamEvent,
 	gh.TeamAddEvent,
+	gh.TeamEvent,
 	gh.WatchEvent,
 }
 
-// SubjectFromGitHubEvent computes the CE subject from GitHub event
-func SubjectFromGitHubEvent(gitHubEvent gh.Event, payload interface{}, logger *zap.SugaredLogger) string {
+// SubjectAndExtensionsFromGitHubEvent computes the CE subject from GitHub event
+func SubjectAndExtensionsFromGitHubEvent(gitHubEvent gh.Event, payload interface{}, logger *zap.SugaredLogger) (string, map[string]interface{}) {
```

Comment:
I would revert this change, and made a new method to extract or directly set the extensions.

Reply:
The reason I put them together, but I agree they arguably shouldn't be, is because different methods would use essentially the same switch statement. I can switch it to two methods.